html/captive-portal/lib/captiveportal/PacketFence/Controller/Activate/Email.pm

```diff
@@ -120,6 +121,7 @@ sub login : Private {
         $c->clear_errors;
     }
     $c->stash(
+        title => "Guest Sponsor Login",
```

Comment:
i18n or directly in the template IMO

The configuration will get heavy for something that is never changed